scripts/s3-logrotate/s3supportbundlefilerollover.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# This script is to make sure we have only fixed number of s3 support bundle files
+# on the jenkins node, at any given time.
+
```

Comment:
please check the file permissions are correct like other log rotate scripts

Reply:
yes, I gave the same file permissions as other files in this directory.